models/Document/TargetingDocument.php

```diff
@@ -103,15 +103,30 @@ public function getTargetGroupSpecificElementNames(): array
      * @param Tag $data
      *
      * @return PageSnippet
+     *
+     * @deprecated since v6.7 and will be removed in 7. Use setEditable() instead.
      */
     public function setElement($name, $data)
```

Comment:
`setElement()` can go here since the alias is already covered in `PageSnippet` 😉

Reply:
good catch 👍 removed it :grin: